src/System.Collections.Immutable/tests/Configurations.props

```diff
@@ -3,6 +3,8 @@
   <PropertyGroup>
     <BuildConfigurations>
       netstandard1.3;
+      netstandard;
+      netcoreapp;
```

Comment:
I would definitely expect this to caused a solution file to be updated when you run an update. It should now prefer the netcoreapp configuration for this test project.